lib/uklibid/infodump.c

```diff
@@ -0,0 +1,50 @@
+#include <uk/libinfo.h>
+#include <uk/ctors.h>
+#include <uk/print.h>
+#include <uk/assert.h>
+
+static void libinfo_bootdump(void)
```

Comment:
Sounds good, also here I thought we can improve after the release because the infrastructure is there by then. Another option that we could do is to dump the library list on a crash screen.

Reply:
Ok 👍🏼